tests/e2e/cucumber/steps/api.ts

```diff
@@ -139,3 +140,33 @@ Given(
     }
   }
 )
+
+Given(
+  '{string} creates the following file(s) in personal space using API',
+  async function (this: World, stepUser: string, stepTable: DataTable): Promise<void> {
+    const user = this.usersEnvironment.getUser({ key: stepUser })
+    for (const info of stepTable.hashes()) {
+      await api.dav.uploadFileInPersonalSpace({
```

Comment:
as this is a Given step and an API step I think we should at least check the reponse code of the upload. If the upload does not work for some reason the test will fail in a later step and that will be very confusing to debug

Reply:
we have a function checkResponseStatus(response, Failed while uploading file '${pathToFile}' in personal space) that is checked while creating the file which is inside createFile function